src/field_theory/adjoin.lean

```diff
@@ -321,42 +328,31 @@ section induction
 
 variables {F : Type*} [field F] {E : Type*} [field E] [algebra F E]
 
+lemma fg_of_noetherian (S : intermediate_field F E)
+  [is_noetherian F E] : ∃ (t : finset E), S = adjoin F ↑t :=
```

Comment:
Is there a reason why you don't phrase this in terms of `subalgebra.fg`?

Reply:
The statement is slightly different since `subalgebra.fg` talks about `algebra.adjoin` and I wanted `intermediate_field.adjoin`. How about splitting off `S.to_submodule.fg → ∃ t, S = intermediate_field.adjoin F t` (and maybe `S.to_submodule = algebra.adjoin F t → S = intermediate_field.adjoin F t`)?

Reply:
(Done: see a94b923)